cmd/runner/executetest.go

```diff
@@ -67,7 +69,7 @@ func indent(level int, format string, a ...any) {
 	fmt.Println()
 }
 
-func executeTest(ctx context.Context, k *kubernetes.Kubernetes, plan *TestPlan) bool {
+func executeTest(ctx context.Context, k *kubernetes.Kubernetes, plan *TestPlan, probeImageFlag string) bool {
```

Comment:
How about renaming this?
```suggestion
func executeTest(ctx context.Context, k *kubernetes.Kubernetes, plan *TestPlan, defaultProbeImage string) bool {
```